image_provisioner/playbooks/roles/docker-spread-images/tasks/main.yaml

```diff
@@ -0,0 +1,24 @@
+---
+- name: Creates directory
+  file:
+    path: "{{ image_dir }}"
+    state: directory
+
+- name: Rsync docker images
+  synchronize:
+    src: "{{ image_dir }}"
+    dest: "{{ image_dir }}"
+  delegate_to: "{{ jump_node_ip }}"
+    
+- name: Register all images
+  find: 
+    path: "{{ image_dir }}"
+    file_type: file
+    recurse: yes
+  register: image_files
+
+- debug: var=image_files
+
+- name: Docker load images
+  command: docker load -i {{ item.path }}
+  with_items: "{{ image_files.files }}"
```

Comment:
I don't think we have.  But remember we're using a pretty high-end rig.  Also "local" isn't always "local".  In our scenario, local is actually network.